resources/event-bus/requirements.yaml

```diff
@@ -0,0 +1,11 @@
+dependencies:
+  - name: publish
+    condition: global.eventBus
```

Comment:
This condition should be `global.kymaEventBus`

Reply:
Done